README.md

```diff
@@ -101,6 +101,9 @@ $ npm run dev
 
 // shell terminal two
 $ npm run serve
+
+// shell terminal three
```

Comment:
Why is this added here? There are no polymer dependencies in this repo?

Reply:
On line 94 it says to open three different shell sessions, with the third to "run the Polymer server".
But I tried running the demo.html without the third shell session running Polymer, and I can view the demo site.

So is the last shell session needed or not?